style.css

```diff
@@ -1,17 +1,51 @@
-nav {
-    position: fixed;
-    width: 100%;
-    top: 0;
-    left: 0;
-  }
-
-#navbar {
-   
-        display: inline;
-      }
```

Comment:
There are still some issues I think you need to improve on. 😄 👋
[ ] Be carefull with the spaces between your code. It's a very important part for pass throw the linters.